v1/ast/capabilities.go

```diff
@@ -97,17 +120,29 @@ func CapabilitiesForThisVersion() *Capabilities {
 		return f.Builtins[i].Name < f.Builtins[j].Name
 	})
 
-	for kw := range futureKeywords {
-		f.FutureKeywords = append(f.FutureKeywords, kw)
-	}
-	sort.Strings(f.FutureKeywords)
+	if co.regoVersion == RegoV0 || co.regoVersion == RegoV0CompatV1 {
+		for kw := range allFutureKeywords {
+			f.FutureKeywords = append(f.FutureKeywords, kw)
+		}
 
-	f.Features = []string{
-		FeatureRefHeadStringPrefixes,
-		FeatureRefHeads,
-		FeatureRegoV1Import,
+		f.Features = []string{
+			FeatureRefHeadStringPrefixes,
+			FeatureRefHeads,
+			FeatureRegoV1Import,
+		}
+	} else {
+		for kw := range futureKeywords {
+			f.FutureKeywords = append(f.FutureKeywords, kw)
+		}
+
+		f.Features = []string{
+			FeatureRegoV1,
+		}
 	}
```

Comment:
The `rego_v1` feature implies support for v0 capabilities and features.

The downside is that older OPAs won't understand this when consuming the capabilities from a 1.0 OPA.
Which means that a v0/v1-compatible policy that both systems understand, like: 

```
package example

import future.keywords

p contains x if {
  x := 42
} 
```

can't be built by the v0 OPA using the v1 OPA's capabilities.